roles/metrics/defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+foreman_metrics_url: "https://{{ ansible_fqdn | default('localhost') }}/metrics"
+foreman_pcp_additional_packages: "{{ ['pcp-system-tools'] if ansible_os_family == 'RedHat' else [] }}"
+foreman_pcp_optional_agents_redhat:
+  - apache
+  - openmetrics
+  - postgresql
+  - redis
+foreman_pcp_optional_agents: "{{ foreman_pcp_optional_agents_redhat if ansible_os_family == 'RedHat' else [] }}"
```

Comment:
this currently means we do not enable those PMDAs on Debian at all, but in reality we only want not to install their packages… it's complicated.